.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Purpose
+_Describe the problem or feature in addition to a link to the issues._
+
+## Approach
+_How does this change address the problem?_
+
+## Testing
+_How did you test your changes?_
+
+#### Open Questions and Pre-Merge TODOs
+- [ ] Use github checklists. When solved, check the box and explain the answer.
+
+## Learning
```

Comment:
I guess this is where we'd cite our StackOverflow and documentation sources

Reply:
Yep :-)